sys/net/network_layer/fib/fib.c

```diff
@@ -111,11 +111,13 @@ static int fib_find_entry(uint8_t *dst, size_t dst_size,
                 if (fib_table[i].global != NULL) {
                     universal_address_rem(fib_table[i].global);
                     fib_table[i].global = NULL;
+                    fib_table[i].global_flags = 0;
```

Comment:
it seems reasonable. Are these the only two places where `global` and `next_hop` can get `NULL`? If not it might make sense to move `fib_table[i].global_flags = 0;` (and the corresponding `next_hop` statement) outside of the enclosing if